ckanext/security/controllers.py

```diff
@@ -168,7 +168,7 @@ def new(self, id=None):
         helpers.flash_success(_('Successfully updated two factor authentication secret. Make sure you add the new secret to your authenticator app.'))
         helpers.redirect_to('mfa_configure', id=user_id)
 
-
+mailer.send_reset_link = secure_mailer.send_reset_link
```

Comment:
:monkey: patch the longer password reset token behaviour onto the CKAN core mailer, this way when we call `original_password_reset` it will use our version of `send_reset_link`.